### DISCUSSION TITLE
请问为什么打包出来的App不能正常使用

### DISCUSSION THREAD

**Discussion by nbwzx** [tw93/Pake#407] [Q&A]
用 GitHub Actions 产生了一个 [App](https://github.com/nbwzx/Pake/actions/runs/4836906884)
截图如下:
![QL(73`BMCR$82O@2}HIHOGM](https://user-images.githubusercontent.com/55343466/235295143-773b80b0-04c6-4862-b893-ca1a51ef341b.png)
其中的元素也都无法点击。

但是[实际网页](https://blddb.net) ([代码](https://github.com/nbwzx/blddb)) 能正常使用：
![IEPM@AN@2P(26 MT2W9ONVA](https://user-images.githubusercontent.com/55343466/235295197-4fb53ea3-fb4b-43e0-b3ca-d91e2cf90be3.png)

**Comment by Chanyon** [comment]
我在本地使用(tauri+vite)成功打包了
